hmp/utils.py

```diff
@@ -1112,6 +1009,7 @@ def event_topo(
     """
     if estimate_method is None:
         estimate_method = "max"
+<<<<<<< HEAD
```

Comment:
Conflict to deal with, 516db5c is correct

Reply:
Done